packages/api-form-builder/__tests__/forms.test.ts

```diff
@@ -290,7 +368,10 @@ describe('Form Builder "Form" Test', () => {
         const { id } = create.data.formBuilder.createForm.data;
 
         // Add fields definitions
-        await updateRevision({ revision: id, data: { fields } });
+        await updateRevision({
+            revision: id,
+            data: { fields, steps: [{ title: "", layout: [] }] }
```

Comment:
Seeing this test here, not only it might be better to have a title here just saying "Test step" at lest, but also maybe have validation that ensures title is not an empty string. Can we try to see if this is easily doable and if so, can we do it?